packages/replay/src/constants.ts

```diff
@@ -42,3 +42,6 @@ export const CONSOLE_ARG_MAX_SIZE = 5_000;
 export const SLOW_CLICK_THRESHOLD = 3_000;
 /* For scroll actions after a click, we only look for a very short time period to detect programmatic scrolling. */
 export const SLOW_CLICK_SCROLL_TIMEOUT = 300;
+
+/** When encountering a total segment size exceeding this size, stop the replay (as we cannot properly ingest it). */
```

Comment:
We're adding up the size of all segments, right? 

```suggestion
/** When encountering a total replay size exceeding this size, stop the replay (as we cannot properly ingest it). */
```